Cargo.toml

```diff
@@ -35,6 +35,7 @@ verbose = []
 "phy-tap_interface" = ["std", "libc"]
 "proto-ipv4" = []
 "proto-igmp" = ["proto-ipv4"]
+"proto-dhcpv4" = ["proto-ipv4", "socket-raw"]
```

Comment:
This line breaks compilation because there is already a feature with that name in line 44. I assume that it's a rebase conflict?